### ISSUE TITLE
output markup 

### ISSUE THREAD

**Issue by williamengbjerg** [Alexeykhr/vue-stripe-menu#70]
Do you have an example for the full html mark-up?

**Comment by diomed** [comment]
https://github.com/Alexeykhr/vue-stripe-menu/issues/48

**Comment by williamengbjerg** [comment]
I was more thinking the way to inject the code from the menu vue. 

`<menu all-attributes>
markup here for generating the menu I general
</menu>`

https://github.com/Alexeykhr/vue-stripe-menu/blob/dependabot/npm_and_yarn/dev/babel/preset-env-7.7.6/src/components/Menu.vue

**Comment by Alexeykhr** [comment]
If I understand the question correctly, that is 4 props (sorry, I just noticed that I forgot to describe another slot in the documentation):

[Vue Stripe Menu - Slots](https://alexeykhr.github.io/vue-stripe-menu/#-menu--slots)

- The first and main one is the **default** slot-scope, which is responsible for displaying content in the dropdown menu. The component goes through each element of the array with the **menu** and passes it to these props.

_Example:_

```
<vsm-menu :menu="menu">
  <template #default="data">
    <!--Dropdown Content-->
  </template>
</vsm-menu>
```

- Next 2 props go as an addition to the menu (if we want to add something of our own). This can be clearly seen from the example that was dropped above - we added the **Logo** on the left and the **Sign In** button on the right. ([Codepen](https://codepen.io/Alexeykhr/pen/YzPKxpX))

_Example:_

```
<vsm-menu :menu="menu">
  <template slot="before-nav">
    <!--Content Left-->
  </template>
  <template slot="after-nav">
    <!--Content Right-->
  </template>
</vsm-menu>
```

- The last slot at the time of writing is for menu items. Those. if you transfer the menu array, then naturally the text will only be displayed, and if we want to push the picture as well, then this can be done through this slot.

_Example:_

```
<vsm-menu :menu="menu">
  <template slot="title" slot-scope="data">
    <div v-if="data.item.title === 'MyTitle'">
      <!--<img src="My additional attribute with image url" :alt="data.item.title">-->
    </div>
    {{ data.item.title }}
  </template>
</vsm-menu>
```

I think later I will add a complete example with all the filled attributes and other
